uber/templates/preregistration/preregbase.html

```diff
@@ -1,35 +1,135 @@
 {% extends "base.html" %}
-{% block backlink %}
-
-
-{% endblock %}
+{% block backlink %}{% endblock %}
 {% block header %}
```

Comment:
Unrelated change; we were previously doing a check every 90 seconds.  I suspect this check sometimes fails and that's why some people complain they can't fill out the form before their browser refreshes.  To mitigate the issue somewhat I updated the check to fire every 15 minutes instead.